### ISSUE TITLE
Add Banner fields to Vet Center content type in Drupal

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#18924]
## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by omahane** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16076 is the VBA work that could be a model.

**Comment by mmiddaugh** [comment]
@Agile6MSkinner we should consider timing - we need to get the business logic nailed down for when it can/should be used and communicate that with editors before this brand new and prominent field is exposed to editors.

**Comment by Agile6MSkinner** [comment]
@mmiddaugh Definitely. This Drupal story will not expose the banner on the front end, but could confuse editors when the see the fields. I'll get together a CM ticket.

**Comment by mmiddaugh** [comment]
@davidmpickett @Agile6MSkinner RCS has approved the addition of banners.  They understand the field will be available to editors but intend to manage it via business process. 
- We'd like to add help text discouraging use by local editors and directing them to contact RCS to add a banner to their page. 
- Ideally, this would be set up in a way that enables reporting/auditing is not required if the lift is too high. 
- KB articles should focus on RCS editor level.
